Hosts are
0: di36
di36
Party variable: helsinki
Traceback (most recent call last):
  File "fig-partisanship-helsinki.py", line 45, in <module>
    df1 = pd.read_csv(inputpath + infilename1, delimiter = ',', lineterminator = "\n")
  File "/home/jernie/new_env/lib/python3.7/site-packages/pandas/io/parsers.py", line 610, in read_csv
    return _read(filepath_or_buffer, kwds)
  File "/home/jernie/new_env/lib/python3.7/site-packages/pandas/io/parsers.py", line 462, in _read
    parser = TextFileReader(filepath_or_buffer, **kwds)
  File "/home/jernie/new_env/lib/python3.7/site-packages/pandas/io/parsers.py", line 819, in __init__
    self._engine = self._make_engine(self.engine)
  File "/home/jernie/new_env/lib/python3.7/site-packages/pandas/io/parsers.py", line 1050, in _make_engine
    return mapping[engine](self.f, **self.options)  # type: ignore[call-arg]
  File "/home/jernie/new_env/lib/python3.7/site-packages/pandas/io/parsers.py", line 1867, in __init__
    self._open_handles(src, kwds)
  File "/home/jernie/new_env/lib/python3.7/site-packages/pandas/io/parsers.py", line 1368, in _open_handles
    storage_options=kwds.get("storage_options", None),
  File "/home/jernie/new_env/lib/python3.7/site-packages/pandas/io/common.py", line 647, in get_handle
    newline="",
FileNotFoundError: [Errno 2] No such file or directory: '/Users/jeremiasnieminen/Dropbox/local_speech/analysis/code/helsinkianalysis/output/helsinki/partisanship-helsinki_Cadj_c0-cis.csv'
srun: error: di36: task 0: Exited with exit code 1
srun: Terminating job step 2495898.1
